src/Symfony/Bundle/FrameworkBundle/Command/YamlLintCommand.php

```diff
@@ -37,6 +37,7 @@ protected function configure()
             ->setDescription('Lints a file and outputs encountered errors')
             ->addArgument('filename', null, 'A file or a directory or STDIN')
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The output format', 'txt')
+            ->addOption('exclude_dir', null, InputOption::VALUE_OPTIONAL, 'Exclude a list of directory', '')
```

Comment:
What about CI in mixed environments... symfony + nodejs?
We have *.yml in a node_module dir, we do not want to lint (too big and it does not belong to our us) them, but we might want to lint all in src/ as well as all in app/, without having to specify a white list, but rather a black list.
the proposed options seams to me pretty reasonable, multiple options... why not, but again... it just makes the command line longer.